actions/release-candidate/dist/main/index.js

```diff
@@ -31950,14 +31985,42 @@ async function run() {
 		fs.appendFileSync(`${ sbt_dir }/plugins/build.sbt`, 'addSbtPlugin("com.github.sbt" % "sbt-pgp" % "2.1.2")\n');
 		fs.appendFileSync(`${ sbt_dir }/build.sbt`, `pgpSigningKey := Some("${ gpg_signing_key_id }")\n`);
 
-		// enable SBT for publishing SBOM
+		// enable SBT for publishing SBOM either locally or remotely
 		fs.appendFileSync(`${ sbt_dir }/plugins/build.sbt`, 'addSbtPlugin("com.github.sbt" %% "sbt-sbom" % "0.4.0")\n');
 		fs.appendFileSync(`${ sbt_dir }/build.sbt`, 'bomFormat := "xml"\n');
 
-		if (publish) {
+		let svn_config_dir = "";
+		if (do_publish) {
+			// if publishing is enabled, we must configure sbt to write to a config file for
+			// post to read from
+			const svn_username = core.getInput("svn_username", { required: true });
+			const svn_password = core.getInput("svn_password", { required: true });
+
+			// Create a temporary config directory
+			svn_config_dir = `${ os.tmpdir }/svn-config`
```

Comment:
Is there file/directory that svn reads by default? It would be nice if users didn't have to know to pass --config /tmp/svn-config in order to do svn commands. Maybe svn reads something like ~/.svn/config?

Reply:
It reads from .subversion without --config-dir option, but it expects very specific files, ex servers, and there is already an existing default...are we ok with overwriting it? It will ignore any files not named accordiing to the what it's expecting so we cant add a servers2 file for example